pypeclub/OpenPype
# Ftrack: Where I run action enhancement

## Issue
Ftrack action `Where I Run` may be helpful to identify where is running second instance of OpenPype using same ftrack username but is not available in OpenPype3.

## Changes
- merged 2 actions handling the discovery of action and showing info into one action
- the action is enabled by default
- the action is in `OpenPype Admin` submenu
- always shows information about all running instances

## How to test
- run tray and trigger action `OpenPype Admin - Where I run`
    - you should see info about the workstation
- if you'll run tray twice you should see 2 informations
